generator/views.py

```diff
@@ -98,6 +100,8 @@ def get(cls, request, share_id):
 
         rom_form = RomForm()
         context = {'share_id': game.share_id,
+                   'is_permalink': True,
+                   'base_uri': request.build_absolute_uri(''),
```

Comment:
This could possibly be better, but it'll pull the hostname out of the HTTP request. Really, it should be grabbing the _canonical_ host, so probably it should be always producing e.g. beta.ctjot.com and not www.beta.ctjot.com. I couldn't figure out if Django had a primitive for that though and this works fine enough anyway.